### PR TITLE
zh-translation: about/supported-releases/index.md

### PR DESCRIPTION
**Pull request by Ruil1n**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/about/supported-releases/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: 支持的版本
```
Comment (kebe7jun) [inline content/zh/about/supported-releases/index.md:2]: 叫`版本支持`是不是好一些？

File: content/zh/about/supported-releases/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+此页面列出了当前支持的版本的状态，时间表和策略。受支持的 Istio 版本包括处于维护窗口期以及为安全问题和错误提供了补丁的版本。Minor 版本中的补丁版本不包含向后的兼容性。
+
+* [支持策略](#support-policy)
+* [命名方案](#naming-scheme)
+* [Istio 版本的支持状态](#support-status-of-istio-releases)
+* [没有通用漏洞披露（CVEs）的版本](#releases-without-known-Common-Vulnerabilities-and-Exposures)
+
+## 支持策略{#support-policy}
+
+每个提交我们会产生 Istio 的新构建。大约每个季度我们会构建一个 Minor 版本并进行大量测试和发行认证。我们会对在 Minor 版本中发现的问题发布补丁版本。
```
Comment (kebe7jun) [inline content/zh/about/supported-releases/index.md:17]:
```suggestion
每个提交会产生 Istio 构建版本。大约每个季度我们会构建一个 Minor 版本并进行大量测试和发行认证。并对在 Minor 版本中发现的问题发布补丁版本。
```

File: content/zh/about/supported-releases/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+不同类型代表不同的产品质量水平和 Istio 团队对其的不同支持水平。在这种情况下，*支持* 意味着我们将为关键问题发布补丁并提供技术支持。另外，第三方和合作伙伴也可能提供长期支持方案。
```
Comment (kebe7jun) [inline content/zh/about/supported-releases/index.md:19]:
```suggestion
不同类型代表不同的产品质量，Istio 团队对其有不同的支持力度。在这种情况下，标明 *支持* 意味着我们将为关键问题发布补丁并提供技术支持。另外，第三方和合作伙伴也可能提供长期支持方案。
```

File: content/zh/about/supported-releases/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+|类型              | 支持级别                                                                                                         | 质量和建议使用场景
+|------------------|-----------------------------------------------------------------------------------------------------------------------|----------------------------
+|开发构建 | 不支持                                                                                                            | 危险，不完全可靠。建议用于实验、测试场景。
+|Minor 版本     | 在下一个 Minor 版本后的 3 个月内提供支持                                                       | 可安全地用于生产环境。建议用户尽快升级到这些版本。
+|补丁             | 与相应的 Minor 版本相同                                                                               | 建议用户在补丁可用时尽快采用。
+|安全补丁    | 与补丁相同，但是除了上一个补丁程序的安全修复之外，它将不包含任何其他代码 | 因为安全修复的重要性，**强烈** 建议用户采用发行版本后的安全补丁。
+
+您可以在[发布页面](https://github.com/istio/istio/releases)上找到可用的版本，如果您是冒险类型，您可以在[每日构建 wiki](https://github.com/istio/istio/wiki/Daily-builds)上了解我们的每日构建。您可以在[此处](/zh/news)找到每个 Minor 版本和补丁版本的高级发行说明。
```
Comment (kebe7jun) [inline content/zh/about/supported-releases/index.md:28]:
```suggestion
您可以在[发布页面](https://github.com/istio/istio/releases)上找到可用的版本，如果您是愿意冒险，可以在[每日构建 wiki](https://github.com/istio/istio/wiki/Daily-builds)上了解我们的每日构建，也可以在[此处](/zh/news)找到每个 Minor 版本和补丁版本的高级发行说明。
```

File: content/zh/about/supported-releases/index.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+| 1.9             | 是                   | 2021年2月9日    | ～2021年8月（预期） | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
+| 1.8             | 是                   | 2020年11月10日   | ～2021年5月（预期） | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
+| 1.7             | 否                    | 2020年8月21日     | 2021年2月25日        | 1.16, 1.17, 1.18              | 1.15                                          |
+| 1.6             | 否                    | 2020年5月21日        | 2020年11月23日   | 1.15, 1.16, 1.17, 1.18        |                                               |
+| 1.5 及更早 | 否                    |                     |                     |                               |                                               |
+
+## 没有通用漏洞披露（CVEs）的版本{#releases-without-known-Common-Vulnerabilities-and-Exposures}
+
+{{< warning >}}
+Istio 不保证超出支持窗口期的 Minor 版本都有已知的 CVEs 补丁。请保持最新并使用受支持的版本。
```
Comment (kebe7jun) [inline content/zh/about/supported-releases/index.md:54]:
```suggestion
Istio 不保证超出支持窗口期的 Minor 版本都有已知的 CVEs 补丁。请使用最新和受支持的版本。
```

File: content/zh/about/supported-releases/index.md (hunk 1 above)
Comment (Ruil1n) [reply content/zh/about/supported-releases/index.md]: 已修正

**Comment by Ruil1n** [comment]
@googlebot I fixed it.

**Comment by kebe7jun** [comment]
@googlebot I fixed it.

**Comment by kebe7jun** [comment]
@Ruil1n plz resolve the lint_istio.io test in https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/9118/lint_istio.io/1367748340682330112

**Review by kebe7jun — APPROVED** [review]

**Review by rootsongjc — APPROVED** [review]
/LGTM

**Pull request merged**